.changelog/3845.breaking.2.md

```diff
@@ -0,0 +1,3 @@
+go/consensus/api/transaction/testvectors: Rename `TestVector`'s `Valid` field
```

Comment:
I guess technically this is breaking for the `testvectors` binary, but thinking about it, would we really bump release version for a breaking change there?

Reply:
Probably not, I would put this under internal.

Reply:
Yeah, I admit this is a grey area and I also thought about adding this under internal category. No strong preferences here.

I think we wouldn't bump the consensus version for this, but at the same time I wouldn't simply add this kind of change in a bugfix release.